mlem/contrib/numpy.py

```diff
@@ -148,10 +157,20 @@ def serialize(self, instance: np.ndarray):
         return instance.tolist()
 
     def _check_shape(self, array, exc_type):
-        if tuple(array.shape)[1:] != self.shape[1:]:
-            raise exc_type(
-                f"given array is of shape: {(None,) + tuple(array.shape)[1:]}, expected: {self.shape}"
+        if self.shape:
```

Comment:
this will check if shape is None or shape is empty tuple. Maybe check if it is None?